pom.xml

```diff
@@ -1109,7 +1109,7 @@
             </dependency>
             <dependency>
                 <groupId>org.apache.nifi</groupId>
-                <artifactId>nifi-kudu-nar</artifactId>
```

Comment:
Not sure why the diff made this a "replace" versus an "append", can you rebase against the latest master branch? If that doesn't take care of the problem, please ensure you are adding your NAR and not replacing someone else's :)

Reply:
This is still an issue with the latest PR